fasttrips/Stop.py

```diff
@@ -207,8 +207,9 @@ def add_daps_tazs_to_stops(self, dap_df, dap_id_colname, taz_df, taz_id_colname)
 
         # write the stop ids and zone ids to numbering file
         stop_id_df = self.stop_id_df  # local copy with filled NA
-        if Stop.STOPS_COLUMN_ZONE_ID_NUM not in stop_id_df.columns.values:
-            stop_id_df[Stop.STOPS_COLUMN_ZONE_ID_NUM] = -1
+        for col in [Stop.STOPS_COLUMN_ZONE_ID_NUM, Stop.STOPS_COLUMN_ZONE_ID]:
```

Comment:
This looks like you added STOPS_COLUMN_ZONE_ID to the list, is that on purpose?

Reply:
Hmmm I remember it failing tests when you don't. Let me try again.

Reply:
Confirmed. This is needed in order to fix test_bunny

Reply:
Hm, four lines down nans are filled with 'None', maybe that's the right way to go here too then?

Reply:
I replaced -1 with NaN, so the behaviour from the line you mentioned is preserved

Reply:
My first thought was "that's not right", but then I realised they will be filled in the next step, so all good :)